### PR TITLE
*: add warnings for experimental features

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What is changed, added or deleted? (Required)

Add warnings for experimental features.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -283,6 +283,10 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 
 prepare 语句的 Plan cache 设置。
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (bb7133) [inline reference/configuration/tidb-server/configuration-file.md:288]: How about rename it to "当前改功能为实验特性，不建议在生产环境中使用。", similar to the existing ones:

https://github.com/pingcap/docs-cn/blob/master/reference/sql/attributes/auto-random.md#auto_random-%E4%BB%8E-v310-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5

Comment (jackysp) [reply reference/configuration/tidb-server/configuration-file.md:288]: ok

Comment (jackysp) [inline reference/configuration/tidb-server/configuration-file.md:288]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/configuration/tikv-server/configuration-file.md [hunk 2]
```diff
@@ -493,6 +493,10 @@ raftstore 相关的配置项。
 + 开启 rocksdb delete_range 接口删除数据的开关。
 + 默认值：false
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (jackysp) [inline reference/configuration/tikv-server/configuration-file.md:498]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/configuration/tikv-server/configuration-file.md [hunk 3]
```diff
@@ -726,6 +730,10 @@ rocksdb 相关的配置项。
 
 Titan 相关的配置项。
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (jackysp) [inline reference/configuration/tikv-server/configuration-file.md:735]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/sql/data-types/json.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ category: reference
 
 # JSON 类型
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (jackysp) [inline reference/sql/data-types/json.md:10]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/sql/functions-and-operators/json-functions.md [hunk 5]
```diff
@@ -6,6 +6,10 @@ aliases: ['/docs-cn/sql/json-functions/']
 
 # JSON 函数及语法糖
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (jackysp) [inline reference/sql/functions-and-operators/json-functions.md:11]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/sql/generated-columns.md [hunk 6]
```diff
@@ -6,6 +6,10 @@ aliases: ['/docs-cn/sql/generated-columns/']
 
 # 生成列
 
+> **警告：**
+>
+> 该功能还在测试阶段，请不要在生产环境中使用。
```
Comment (jackysp) [inline reference/sql/generated-columns.md:11]:
```suggestion
> 当前改功能为实验特性，不建议在生产环境中使用。
```

File: reference/configuration/tidb-server/configuration-file.md (hunk 1 above)
Comment (jackysp) [reply reference/configuration/tidb-server/configuration-file.md]: PTAL

File: reference/configuration/tikv-server/configuration-file.md [hunk 7]
```diff
@@ -726,6 +730,10 @@ rocksdb 相关的配置项。
 
 Titan 相关的配置项。
 
+> **警告：**
+>
+> 当前该功能为实验特性，不建议在生产环境中使用。
```
Comment (zhangjinpeng1987) [inline reference/configuration/tikv-server/configuration-file.md:735]: 不建议在 mission-critical 的生产环境使用？

Comment (jackysp) [reply reference/configuration/tikv-server/configuration-file.md:735]: 给一个完整说法吧。

Comment (jackysp) [inline reference/configuration/tikv-server/configuration-file.md:735]:
```suggestion
> 当前该功能为实验特性，不建议在严苛生产环境中使用。
```

**Pull request merged**